src/instructlab/sdg/pipelines/simple/mmlu_bench.yaml

```diff
@@ -0,0 +1,15 @@
+version: "1.0"
+blocks:
+  - name: gen_mmlu_knowledge
```

Comment:
In aakankshaduggal/sdg#15 this is removed from from "full" knowledge pipeline - search for MMLUBenchFlow. So I'd expect this block to be removed from `full/knowledge.yaml`